### PR TITLE
Set fixed ip to local cluster

### PR DESCRIPTION
**Pull request by arikmaor**
Also adds a `zsh` function to setup `kubeconfig` to point to `microk8s`
I've added this because is suddenly stopped working on the laptop

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,10 @@ vim $HOME/.zshrc
 # alias kctx=kubectl
 #
 # autoload -U compinit && compinit
+#
+# set_microk8s_config() {
+#   microk8s config > $HOME/.kube/config
+# }
```
Comment (AsafYehezkelTL) [inline README.md:39]: is it environment agnostic?

**Review by yotamazriel — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -68,6 +72,8 @@ sudo snap install microk8s --classic
 sudo usermod -a -G microk8s $USER
 sudo chown -f -R $USER ~/.kube
 microk8s enable ingress dns storage rbac gpu
+microk8s enable host-access:ip=10.0.1.20
+# add a line `10.0.1.20<TAB>tensorleap.local` to `/etc/hosts`
```
Comment (AsafYehezkelTL) [inline README.md:76]: what does it solve?

**Review by AsafYehezkelTL — APPROVED** [review]
LGTM

**Pull request merged**